slackline.c

```diff
@@ -26,6 +26,9 @@
 #include "slackline.h"
 #include "util.h"
 
+/* CTRL+W: stop erasing if certain characters are reached. */
+#define STOP(c) (!!strchr("\f\n\r\t\v (){}[]\\/#,.)", (c)))
```

Comment:
why double not?

Reply:
It's generally not necessary but a personal preference of mine. With this approach the result will always be either a `1` or a `0`. I don't have strong opinions on that, so if you prefer a single `!`, I'll rebase.

Reply:
Do this..

1. put the string of word breaking characters into a macro constant
3. call ... && strchr(WORD_BREAK_CHAR, *(sl->ptr-1)) != NULL

..or that..

1. turn the "!!" into "!= NULL" or leave it blank
2. rename the macro into ISWORDBREAKER()

feel free to use some handier names for the macros.

Reply:
btw. I forgot to mention that the ")" character occurs two times in your string.

Reply:
rebased. It should be fine now, I also added some more characters.